test/src/stores/tracesStore.spec.js

```diff
@@ -63,6 +63,9 @@ describe('TracesSearchStore', () => {
 
     beforeEach(() => {
         server = new MockAdapter(axios);
+        // Mocking an empty response for the timeline API.
+        // TODO: Change to actual tests.
+        server.onGet(/^\/api\/traces\/timeline\?/g).reply(200, []);
```

Comment:
This server stubs the timeline api endpoint, which resolves an unhandled promise rejection error (the mounted ui calls the timeline endpoint).